PRIVACY_MODEL.md

```diff
@@ -0,0 +1,61 @@
+This proposal introduces new metaprogramming concepts on private fields and methods. This document describes the invariants, guarantees and scope of this privacy as a whole, in light of this feature.
+
+## Private fields and methods are *hidden*
```

Comment:
I don’t personally find the term “hidden” clear; hidden things can be found, private things can’t.

Reply:
I'd like to avoid the term "hard private" due to both the concerns @allenwb raised, and the fact that it sounds a little funny. Do you have an idea for another term?

Reply:
It’s unfortunate that other languages have ruined “private” with “not actually private”, because tbh that’s really the best term.

How about going the verbose route: “unobservable and inaccessible”?

Reply:
How would "inaccessible" be? I think we need to create a piece of jargon that we can make use of easily.

Reply:
hmm, i dunno - "phantom", like a phantom limb?